app/graphql/types/fees/object.rb

```diff
@@ -21,6 +21,7 @@ class Object < Types::BaseObject
       field :taxes_amount_cents, GraphQL::Types::BigInt, null: false
       field :taxes_rate, GraphQL::Types::Float, null: true
       field :units, GraphQL::Types::Float, null: false
+      field :precise_unit_amount, GraphQL::Types::Float, null: false
```

Comment:
GraphQL/OrderedFields: Fields should be sorted in an alphabetical order within their section. Field `precise_unit_amount` should appear before `units`.